src/screens/ProfessionScreen.tsx

```diff
@@ -115,6 +123,15 @@ const ProfessionScreen = ({ navigation }: ProfessionScreenPropsType): JSX.Elemen
     })
   }
 
+  if (error) {
+    return (
+      <View style={styles.root}>
+        <StatusBar backgroundColor='blue' barStyle='dark-content' />
+        <Text>{error}</Text>
+      </View>
+    )
+  }
+
```

Comment:
This leads to the behaviour, that if on SubProfession-Screen, disabling Network, go back to ProfessionScreen, the complete Screen is gone, although it could actually be displayed due to caching.

If just adding <Text>{error]</Text> after </Loading> in Profession-Screen, the hole content is shown, and error message at the bottom. 


Also for the SubCategoryProfessionScreen, if just adding <Text>{error}</Text> right unter </Loading>, the user still gets the information which Profession (s)he tried to load.